profiles/common/themes/ec_resp/theme-settings.php

```diff
@@ -29,4 +29,14 @@ function ec_resp_form_system_theme_settings_alter(&$form, &$form_state) {
     '#title' => t('Enable Interinstitutional theme'),
     '#default_value' => theme_get_setting('enable_interinstitutional_theme'),
   );
+
+  $form['#submit'][] = 'ec_resp_theme_settings_submit';
+}
+
+/**
+ * Form submission handler for ec_resp_form_system_theme_settings_alter().
+ */
+function ec_resp_theme_settings_submit($form, &$form_state) {
+  $separator = ($form_state['values']['enable_interinstitutional_theme'] == 0) ? '>' : ' ';
+  variable_set('easy_breadcrumb-segments_separator', $separator);
```

Comment:
I don't understand why this is needed. Please document explaining why this variable needs to be set.